mne/io/egi/egimff.py

```diff
@@ -745,14 +743,14 @@ def read_evokeds_mff(fname, condition=None, channel_naming='E%d',
 
     Notes
     -----
```

Comment:
I"m not sure if the Notes sectino needs to be removed now

Reply:
From Eric's comment it looks like the `.. versionadded::` tag goes in the notes section.